README.md

```diff
@@ -251,6 +251,11 @@ when initializing your instance of the SplunkRum API:
 - `diskBufferingEnabled(boolean)` : 
   Enables the storage-based buffering of telemetry. 
   This setting is useful when instrumenting applications that might work offline for extended periods of time.
+- `enableSessionBasedSampling(double)` :
+  Enable session ID based sampling and set its sampling ratio. The ratio is a probability of a
+  session being included between between 0.0 (all dropped) and 1.0 (all included). Sampling is only
+  applied if the spans created via `SplunkRum` APIs are root spans. If a custom parent span context
+  was created, then the sampling flag of the parent span is respected instead.
```

Comment:
Is the part about root spans and parent spans still correct after removed the `parentBased` sampling?
```suggestion
  session being included between between 0.0 (all dropped) and 1.0 (all included).
```

Reply:
Removed that part.